.github/workflows/storybook.yml

```diff
@@ -39,7 +39,7 @@ jobs:
                   ssh-private-key: ${{ secrets.KHAN_ACTIONS_BOT_SSH_PRIVATE_KEY }}
 
             - name: Publish to Chromatic
-              uses: chromaui/action@latest
+              uses: chromaui/action@v11
```

Comment:
**note:** Best to pin to a known version to avoid unexpected breaking changes that a major update might introduce.

Reply:
TBH, for a service like this, I'd rather us be using `@latest` and deal with issues if they come up.  Otherwise we tend to live on aging versions. I view this similar to how we live on evergreen browsers these days. Not a blocker for this PR, but I'd be in favour of keeping `@latest`.

Reply:
The problem with `@latest` is it opts us in to protentially breaking changes. We should be in control of that.